src/api.js

```diff
@@ -57,4 +57,31 @@ const patchVotesById = (article_id, inc_votes) => {
   }
 };
 
-export { getArticles, getArticleById, getCommentByArticleById, patchVotesById };
+const postComment = (article_id, body) => {
+  const comment = {
+    body: body,
+    username: "grumpy19",
+  };
+  if (article_id) {
```

Comment:
does an error not get thrown by your backend API if article_id is undefined/not found/invalid? might not need this check as it should be caught in the catch block if so